src/main/java/org/coffeetrain/doggifs/MainActivity.kt

```diff
@@ -5,15 +5,19 @@ import android.content.Context
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.Window
-import android.view.WindowManager
-import flow.*
```

Comment:
I think I set it crazy high in the Square code style file-- higher than the UI would let me.